mlflow/store/db_migrations/versions/7f2a7d5fae7d_add_datasets_inputs_input_tags_tables.py

```diff
@@ -0,0 +1,79 @@
+"""add datasets inputs input_tags tables
+
+Revision ID: 7f2a7d5fae7d
+Revises: 3500859a5d39
+Create Date: 2023-03-23 09:48:27.775166
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from mlflow.store.tracking.dbmodels.models import SqlDataset, SqlInputTag, SqlInputs
+
+# revision identifiers, used by Alembic.
+revision = "7f2a7d5fae7d"
+down_revision = "3500859a5d39"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        SqlDataset.__tablename__,
+        sa.Column("dataset_uuid", sa.String(length=32), nullable=False),
```

Comment:
The length defined in the `models.py` file is 36, but it's 32 here. Can we make this 36?